CHANGELOG

```diff
@@ -1,4 +1,9 @@
 * Wed Dec 07 2016 Trevor Vaughan <tvaughan@onyxpoint.com> - 2.1.0-0
+- Added a set of Data Types
+  - Port (any of the below)
+  - RandomPort (0)
+  - PrivilegedPort (1-1024)
+  - UnprivilegedPort (1025-65535)
```

Comment:
According to [RFC 6335, Section 6](https://tools.ietf.org/html/rfc6335#section-6), `PrivilegedPort` should be `SystemPort` and `UnprivilegedPort` should be `UserPort`. Also, User Ports only go up to 49151; from 49152 to 65535 are "Dynamic Ports." You could probably leave out the dynamic ports for now, if time is an issue.